lib/rules/aria-input-field-name.json

```diff
@@ -2,7 +2,7 @@
   "id": "aria-input-field-name",
   "selector": "[role=\"combobox\"], [role=\"listbox\"], [role=\"searchbox\"], [role=\"slider\"], [role=\"spinbutton\"], [role=\"textbox\"]",
   "matches": "no-naming-method-matches",
-  "tags": ["cat.aria", "wcag2a", "wcag412", "ACT"],
+  "tags": ["cat.aria", "wcag2a", "wcag412", "ACT", "TTv5", "TT5.c"],
```

Comment:
Ha! I would have totally picked 5.A on that one, but you're right. Accessible name, not label.